index.html

```diff
@@ -0,0 +1,129 @@
+                            <dt>Без подзарядки</dt>
+                            <dd>48 часов</dd>                            
+                        </dl>            
+                    </div>
+                    <div class="slider_item">
+                        <img src="#" alt="квадрокоптер">
+                        <p>03</p>
+                        <p class="slider_item">Пархает как бабочка, жалит как пчела!</p>
+                        <p class="slider_item">Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+                        <a class="slider_item" href="#">подробнее</a>
+                        <dl class="slider_item">
+                            <dt>Дальность полета</dt>
+                            <dd>800 м</dd>
+                            <dt>Радиус поражения</dt>
+                            <dd>50 м</dd>                            
+                        </dl>            
+                    </div>
+                </div>                
+            </section>
+            <section>
+                <h2>Популярные товары</h2>
+                <div>
+                    <ul>
+                        <li><a href="#"><img src="#">Виртуальная реальность</a></li>
+                        <li><a href="#"><img src="#">Моноподы для селфи</a></li>
+                        <li><a href="#"><img src="#">Экшн-камеры</a></li>
+                        <li><a href="#"><img src="#">Фитнес-браслеты</a></li>
+                        <li><a href="#"><img src="#">Умные часы</a></li>
+                        <li><a href="#"><img src="#">Квадрокоптеры</a></li>
+                    </ul>                       
+                </div>
+            </section>
+            <section class="service">
+                <h2>Раздел услуг</h2>
+
+
+            </section>
+            <div class="logotypes">
+                <img src="#" alt="DJI logo">
+                <img src="#" alt="SP-Gadgets logo">
+                <img src="#" alt="GoPro logo">
+                <img src="#" alt="VIVE logo">
+            </div>
+            <section class="about-us">
+                <h2>О нас</h2>
+                <p>Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.</p>
+                <p>Мы можем доставить ваш товар в самые отдаленные точки России! DEVICE работает со многими транспортными компаниями</p>
+                <ul class="dilivery_list">
+                    <li>Деловые линии</li>
+                    <li>Автотрейдинг</li>
+                    <li>ЖелДорЭкспедиция</li>
+                </ul>
+                <a href="#">Подробнее о нас</a>
+            </section>
+            <section class="contacts">
+                <h2>Контакты</h2>
+                <p>Вы можете забрать товар сами, заехав в наш офис. Заодно, вы сможете проверить работоспособность покупки. Всякое бывает.</p>
+                <div id="mapcontainer" class="map">
```

Comment:
you'll change this markdown a little, if decide to implement interactive map, istead of static